### PR TITLE
Add setup and setup-this subcommands

### PR DESCRIPTION
**Pull request by barrucadu**
'setup' takes a list of app names, so you can do this:

    govuk-docker setup content-publisher government-frontend

### CODE REVIEW

File: bin/govuk-docker [hunk 1]
```diff
@@ -32,6 +34,20 @@ function check_stack {
...
+    set -x
+    cd $GOVUK_DOCKER_DIR
+    for GOVUK_DOCKER_SERVICE in "$@"; do
+      make $GOVUK_DOCKER_SERVICE
+    done;;
+  "setup-this")
+    check_service
+    set -x
+    cd $GOVUK_DOCKER_DIR
+    make $SERVICE;;
```
Comment (benthorner) [inline bin/govuk-docker]: Is that a typo here?

Comment (barrucadu) [reply bin/govuk-docker]: Where?

**Comment by barrucadu** [comment]
I think there should be a command to set up an app by name, because otherwise you need to be in a directory with the right name, or be in the govuk-docker directory and run `make`.  And it's simple enough to make that command take a list of arguments.

**Comment by tijmenb** [comment]
What would be the difference between `build` and `setup`?

**Comment by barrucadu** [comment]
`build` just does the docker compose bit, `setup` clones the repositories, does the build, and runs setup tasks.  So `build` is a subset of `setup`.

**Comment by tijmenb** [comment]
Worth documenting when to use which one! I'm getting a bit confused already.

**Comment by benthorner** [comment]
Yeah. I think we can remove 'build-this'. We already have a command to setup an app by name - `make app-name` - so I'm keen that we only add one thing to the CLI to solve a given problem. The CLI still worries me, as it's only meant to be a lightweight wrapper - the more stuff we add - the more confusing the environment is likely to become, as it'll become harder to differentiate between docker/compose/make/CLI.

File: bin/govuk-docker (hunk 1 above)
Comment (benthorner) [reply bin/govuk-docker]: We seem to have two semicolons - is that necessary?

Comment (barrucadu) [reply bin/govuk-docker]: Yes, it's how to end a `case` branch (the other branches do this too)

**Comment by barrucadu** [comment]
> We already have a command to setup an app by name - `make app-name` - so I'm keen that we only add one thing to the CLI to solve a given problem

But that requires you to be in the govuk-docker directory, whereas this just requires `govuk-docker` to be in your path.

**Comment by benthorner** [comment]
That's true. What I'm trying to get at, which should also be in the PR description, is: what specific problem(s) / need(s) are we trying to address/support? 

If we're clear about that, it would be easier to understand the change. Currently, this PR adds 3 new commands, if you count the looping behaviour:

   - `govuk-docker setup app-name`
   - `govuk-docker setup app-name...`
   - `govuk-docker setup-this`

It would be helpful if we could document the need devs have for each of these three variants, so that future devs will understand and preserve these commands.

**Comment by barrucadu** [comment]
This is outdated as the `govuk-docker` script has been rewritten in Ruby

**Pull request closed without merging**